catalogue/webapp/components/IIIFViewer/ViewerTopBar.tsx

```diff
@@ -18,6 +18,12 @@ import {
   gridView,
   singlePage,
 } from '@weco/common/icons';
+import { queryParamToArrayIndex } from './IIIFViewer';
+import {
+  getDownloadOptionsFromImageUrl,
+  getDigitalLocationOfType,
+} from '../../utils/works';
+import useTransformedIIIFImage from '../../hooks/useTransformedIIIFImage';
```

Comment:
mega nitpick; shall we go with absolute imports (which would also possibly facilitate the mega-repo switch)?